scripts/userstyles.yml

```
@@ -346,6 +346,15 @@ userstyles:
       app-link: "https://github.com"
     current-maintainers: [*uncenter]
     past-maintainers: [*unseen-ninja, *pocco81, *glowingumbreon]
+  gitlab:
+    name: Gitlab
+    categories: [ development, productivity ]
+    icon: gitlab
+    color: text
```

```suggestion
    color: peach
```
Please please please follow the directions! This should match the color of the brand on https://simpleicons.org/, so orange -> peach.